### PR TITLE
LB load new util

### PR DESCRIPTION
**Pull request by anton-sidelnikov**

### CODE REVIEW

File: csm_test_utils/loadbalancer/lb_monitor.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+AGP = sub_parsers.add_parser("lb_load", add_help=False, parents=[base_parser])
+
+
+def main():
+    args, _ = AGP.parse_known_args()
+    setup_logger(LOGGER, "lb_load", log_dir=args.log_dir, log_format="[%(asctime)s] %(message)s")
+    timeout = 20
+    try:
+        res = requests.get(args.target, headers={"Connection": "close"}, timeout=timeout)
+    except Exception as Ex:
```
Comment (outcatcher) [inline csm_test_utils/loadbalancer/lb_monitor.py:24]: Please use python naming convention (var name should be `snake_case`)

File: csm_test_utils/loadbalancer/lb_monitor.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    else:
+        result = {
+            "reason": LB_TIMING,
+            "client": socket.gethostname(),
+            "server": res.headers["Server"],
+            "elapsed": res.elapsed.microseconds / 1000
+        }
+
+    print(result)
+    exit(0)
```
Comment (outcatcher) [inline csm_test_utils/loadbalancer/lb_monitor.py:41]: There is no need for this exit

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: csm_test_utils/loadbalancer/lb_monitor.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    setup_logger(LOGGER, "lb_load", log_dir=args.log_dir, log_format="[%(asctime)s] %(message)s")
+    timeout = 20
+    try:
+        res = requests.get(args.target, headers={"Connection": "close"}, timeout=timeout)
+    except Exception as Ex:
+        LOGGER.exception("Timeout sending request to LB")
+        result = {
+            "reason": LB_TIMEOUT,
+            "client": socket.gethostname(),
+            "timeout": timeout * 1000,
```
Comment (lego963) [inline csm_test_utils/loadbalancer/lb_monitor.py:29]: do we need this `timeout` field?

**Review by lego963 — CHANGES_REQUESTED** [review]

File: csm_test_utils/loadbalancer/lb_monitor.py (hunk 3 above)
Comment (anton-sidelnikov) [reply csm_test_utils/loadbalancer/lb_monitor.py:29]: yes

File: csm_test_utils/loadbalancer/lb_monitor.py (hunk 1 above)
Comment (anton-sidelnikov) [reply csm_test_utils/loadbalancer/lb_monitor.py:24]: okay)

**Review by lego963 — DISMISSED** [review]

**Review by lego963 — APPROVED** [review]

**Comment by lego963** [comment]
recheck

**Review by outcatcher — DISMISSED** [review]

**Comment by anton-sidelnikov** [comment]
regate

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Comment by outcatcher** [comment]
recheck

**Review by outcatcher — DISMISSED** [review]

**Comment by anton-sidelnikov** [comment]
recheck

**Review by lego963 — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**